src/UI/Settings/Notifications/Edit/NotificationEditView.js

```diff
@@ -33,6 +34,41 @@ var view = Marionette.ItemView.extend({
             model    : this.model,
             property : 'tags'
         });
+
+        if ('PushBullet' === this.model.get('implementation')) {
+            this._ifPushbullet();
+        }
+    },
+
+    _ifPushbullet : function() {
```

Comment:
this is the base view, I'm not sure if we want to have pushbullet specific code in here. the alternative would be to just send all the devices back from the server.